src/radiation/cooling_function.cpp

```diff
@@ -71,6 +73,113 @@ TaskStatus CoolingFunctionCalculateFourForce(MeshBlockData<Real> *rc, const doub
         v(Gye, k, j, i) = 0.;
       });
 
+  // Light Bulb with Liebendorfer model
+  auto &coords = pmb->coords;
+  const bool do_liebendorfer = rad->Param<bool>("do_liebendorfer");
+  const bool do_lightbulb = rad->Param<bool>("do_lightbulb");
+  const Real lum = rad->Param<Real>("lum");
+  if (do_lightbulb) {
```

Comment:
For future reference, these changes should have been made in a separate branch to keep changesets self-contained. It's fine for now though.